x-pack/plugins/security_solution/scripts/run_cypress/parallel.ts

```diff
@@ -70,8 +68,19 @@ ${JSON.stringify(argv, null, 2)}
       const cypressConfigFilePath = require.resolve(`../../${argv.configFile}`) as string;
       const cypressConfigFile = await import(cypressConfigFilePath);
 
-      if (cypressConfigFile.env?.TOOLING_LOG_LEVEL) {
-        createToolingLogger.defaultLogLevel = cypressConfigFile.env.TOOLING_LOG_LEVEL;
+      // Adjust tooling log level based on the `TOOLING_LOG_LEVEL` property, which can be
+      // defined in the cypress config file or set in the `env`
+      {
```

Comment:
It might be useful to have the same functionality for Serverless i.e. the same changes are required in `x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts`.

Have you considered creation a separate file/function like `createLogger()` reading env variable under the hood?

Reply:
Ohhhh. Did not realize there was a `serverless` parallel runner. I will break this out into a utility and will call it from both parallel runners. Thanks for the feedback